### ISSUE TITLE
Issue for training on ImageNet1k

### ISSUE THREAD

**Issue by Aoife-YB** [megvii-research/mdistiller#70]
I configured the training with res34_res18 using the relevant parameters as specified in the appendix. However, the test accuracy was very low. After training for 5 epochs with a learning rate warm-up, the performance was still poor and there were signs of overfitting (the accuracy fluctuated around 28 from epoch 30 to 40). Could it be that the learning rate should be reduced (the configuration in the paper is batch_size=512, lr=0.2) or the T value should be increased (T=1.0)?
